recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
I don't get it. In that case the binary would go to the root directory of the environment, which should not be in the PATH. Isn't that the issue that Kyle points out in #1088?